web.tf

```diff
@@ -23,7 +23,7 @@ resource "aws_security_group" "acme_web" {
     from_port   = 80
     to_port     = 80
     protocol    = "tcp"
-    cidr_blocks = ["10.0.0.0/24"]
```

Comment:
**Accurics remediation**
 
**[HIGH]: Security Groups - Unrestricted Specific Ports - SSH,22**
 
**Vulnerability**
Security Groups - Unrestricted Specific Ports - SSH,22
 
**Remediation**
Change cidr_block to non 0.0.0.0/0

source: accurics-app